ROADMAP.md

```diff
@@ -3,18 +3,30 @@
 This document defines the roadmap for Chaos Mesh development.
 
 ## v1.0
+
 - [x] Support time skew chaos. Simulate time jumping forward or backward.
 - [x] Add container kill chaos. Simulate killing a specified container in a multi-container pod.
 - [x] Add CPU chaos. Simulate CPU being busy.
 - [x] Add memory chaos. Simulate memory allocation failure.
```

Comment:
"Simulate memory allocation failure"

Do we support it?

Reply:
we can use `kernelChaos` to achieve it.